modules/polling/helpers/getRelayerBalance.ts

```diff
@@ -0,0 +1,28 @@
+import { getArbitrumGoerliTestnetSdk, getArbitrumOneSdk } from '@dethcrypto/eth-sdk-client';
+import { config } from 'lib/config';
+import { ArbitrumSdkGenerators } from 'modules/web3/types/contracts';
+import { SupportedNetworks } from 'modules/web3/constants/networks';
+import { DefenderRelayProvider, DefenderRelaySigner } from 'defender-relay-client/lib/ethers';
+import { formatEther } from 'ethers/lib/utils';
+
+export const relayerCredentials = {
+  mainnet: { apiKey: config.DEFENDER_API_KEY_MAINNET, apiSecret: config.DEFENDER_API_SECRET_MAINNET },
+  goerli: { apiKey: config.DEFENDER_API_KEY_TESTNET, apiSecret: config.DEFENDER_API_SECRET_TESTNET }
```

Comment:
maybe we should reuse the same relayer credentials from `getArbitrumPollingContractRelayProvider`

Reply:
💯